src/cluster.c

```diff
@@ -3869,7 +3869,20 @@ void clusterLogCantFailover(int reason) {
         break;
     }
     lastlog_time = time(NULL);
-    serverLog(LL_WARNING,"Currently unable to failover: %s", msg);
+    serverLog(LL_NOTICE,"Currently unable to failover: %s", msg);
+    
+    static int last_vote;
+    static int last_quorum;
+    int cur_vote = server.cluster->failover_auth_count;
+    int cur_quorum = (server.cluster->size / 2) + 1;
```

Comment:
It would be a good idea to move this calculation into a separate function since this is used in several places in this file.